Libraries/Experimental/SwipeableRow/SwipeableQuickActions.js

```diff
@@ -25,11 +30,9 @@ const View = require('View');
  *   <SwipeableQuickActionButton {..props} />
  * </SwipeableQuickActions>
  */
-class SwipeableQuickActions extends React.Component<{style?: $FlowFixMe}> {
-  static propTypes = {
-    style: DeprecatedViewPropTypes.style,
-  };
-
+class SwipeableQuickActions extends React.Component<
+  SwipeableQuickActionsProps,
+> {
   render(): React.Node {
     // $FlowFixMe found when converting React.createClass to ES6
```

Comment:
While typing this component, can you clean up these FlowFixMes as well, as best as possible. This one for example is because children isn't defined in the props: `children: React.Node`